src/apps/filters/h3.c

```diff
@@ -90,7 +90,12 @@ struct Subcommand {
 #define DISPATCH_SUBCOMMAND()                                                \
     for (int i = 0; i < sizeof(subcommands) / sizeof(subcommands[0]); i++) { \
         if (has(subcommands[i].name, 1, argv)) {                             \
-            return subcommands[i].subcommand(argc, argv);                    \
+            H3Error err = subcommands[i].subcommand(argc, argv);             \
+            if (err > 0) {                                                   \
```

Comment:
```suggestion
            if (err != 0) {                                                   \
```

Reply:
I don't have a problem with doing this, but

```c
typedef uint32_t H3Error;
```

since H3Error is an unsigned integer, isn't this the same thing?

Reply:
it looks like on clang and GCC they do indeed compile the same: https://godbolt.org/z/vjPTqanPs

Reply:
Nice to be validated here. :) But I changed it anyway.